plugins/ublox/mm-modem-helpers-ublox.c

```diff
@@ -361,6 +361,8 @@ static const MMModemMode ublox_combinations[] = {
     ( MM_MODEM_MODE_2G | MM_MODEM_MODE_3G | MM_MODEM_MODE_4G ),
     ( MM_MODEM_MODE_2G |                    MM_MODEM_MODE_4G ),
     (                    MM_MODEM_MODE_3G | MM_MODEM_MODE_4G ),
+    (                                       MM_MODEM_MODE_4G ),
```

Comment:
How are these 2 lines with just `MM_MODEM_MODE_4G` not the same as line 360?

Reply:
This change was added from `v1.18`. This was done for the `SARA-R4` series modems. As this change was done in the common helper file so I added this change as it is here. However this does not impact our ublox modems and we can remove it.

Let me know your thoughts on this.

Reply:
It is fine to leave it.  It just looked like a mistake to me.